CONTRIBUTING.md

```diff
@@ -241,7 +241,9 @@ Must be one of the following:
   semi-colons, etc)
 * **refactor**: A code change that neither fixes a bug nor adds a feature
 * **perf**: A code change that improves performance
-* **test**: Adding missing tests
+* **test**: Adding missing tests or correcting existing tests
+* **build**: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
```

Comment:
Now `build` sounds almost identical to `chore`. If we decide to bring this over from Angular 2, we should update other types accordingly.